### ISSUE TITLE
Working with Distributed messaging service like NSQ in Go

### ISSUE THREAD

**Issue by souvikhaldar** [arschles/go-in-5-minutes#163]

**Comment by arschles** [comment]
@souvikhaldar do you have a use case in mind for this?

**Comment by souvikhaldar** [comment]
Yes. Let's say I have a service for clustering of locations, so instead of server doing that I'll let server just intake the location request, send command to the clustering service to process the location input, in the meanwhile the server will respond with an OK status.

**Comment by arschles** [comment]
Thanks @souvikhaldar !
